src/components/CommonComponents/SearchBar/index.tsx

```diff
@@ -68,6 +68,16 @@ const SearchBar = (): JSX.Element => {
     [styles.expanded]: isExpanded,
   });
 
+  const onBlurHandler = (e: React.FocusEvent<HTMLInputElement>) => {
+    if (e.currentTarget.contains(e.relatedTarget)) {
```

Comment:
I might be understanding this logic incorrectly, but it looks like `collapseContainer` could always be called regardless of this if statement? What happens if you remove the whole if statements?

Or if you did something like:

```jsx
if (e.currentTarget.contains(e.relatedTarget) && !isEmpty) {
  return;
}

collapseContainer();
```

Reply:
thanks it's work like description i written

Reply:
let me show you video if i remove the whole if statements.

https://user-images.githubusercontent.com/49237205/197331435-1c14408e-3e35-487e-8607-3364595bfc98.mov

it can't select search list using tab key.

Reply:
Gotcha. What if you change to the statement I mentioned above?

Reply:
Oh you did!